src/main/java/com/appdynamics/extensions/postgres/column/ColumnGenerator.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2019 AppDynamics,Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.appdynamics.extensions.postgres.column;
+
+import com.appdynamics.extensions.postgres.PostgresMonitor;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.appdynamics.extensions.postgres.util.Constants.COLUMNS;
+
+public class ColumnGenerator {
+
+    public static List<Column> getColumnsPOJO(List<Map<String, ?>> columns) {
+        final ObjectMapper mapper = PostgresMonitor.getObjectMapper();
+        Map<String, List<Map<String, ?>>> colAsMap = new HashMap<>();
+        colAsMap.put(COLUMNS, columns);
+        final Columns cols = mapper.convertValue(colAsMap, Columns.class);
+        return cols.getColumns();
```

Comment:
could we add some logging here, to ensure that columns defined in config.yml are properly picked